packages/twenty-front/src/App.tsx

```diff
@@ -58,13 +60,15 @@ export const App = () => {
   const { pathname } = useLocation();
   const pageTitle = getPageTitleFromPath(pathname);
 
+  const { loading: isPrefetchLoading } = useContext(PrefetchContext);
```

Comment:
We should not react to isPrefetchLoading here as this will trigger re-render of the whole app.
It should be loaded in the lowest places that needs it so only this part is re-rendered

Reply:
@gitstart-twenty see my latest comment before treating review feedbacks. I think we should not check the prefetch loading state but the metadata and currentUser queries loading states